src/opds.js

```diff
@@ -554,7 +554,7 @@ var OpdsClient = class OpdsClient {
         const widget = this._widget
         const toplevel = widget instanceof Gtk.Widget
             ? widget.get_toplevel() : null
-        return promptAuthenticate(req, this.username, this.password, toplevel)
+        return promptAuthenticate(req, toplevel)
```

Comment:
I think the `this.username, this.password` bit is still needed for URLs like `http://username:password@example.com/`

Reply:
Yea, looks like you're right. The auth dialog pops up unnecessarily with that kind of URL. Will fix.

Reply:
Should be fixed now.